envwrapper

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+TEMP=$(cat /tmp/dub_dmd_bin_path)
+export DUB_DMD_BIN_PATH=$(TEMP:+:)
+$*
```

Comment:
You might want to `exec` here (and add newline) so that signals are passed down properly

Reply:
That file isn't used anymore. I had forgotten to remove. Needs another approval to run CI again.